api/v1alpha1/openapi.yaml

```diff
@@ -1925,15 +2194,13 @@ components:
           type: string
           description: 'Status of the operation. One of: "Success" or "Failure". More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status'
       description: Status is a return value for calls that don't return other objects.
-    UnprocessableEntityResponse:
-      description: UnprocessableEntityResponse is returned when the request is not valid.
-      type: object
+    Error:
+      required:
+        - message
       properties:
-        error:
+        message:
           type: string
-          description: 'Error message.'
-      required:
-        - error
+          description: Error message
```

Comment:
Nice, now all of them can return proper errors.

We should also talk to the UI folks, sometimes UI expects a list of parameters where the error is detected so they can highlight it on the UI when some input is wrong.

I am pinging them for feedback on that (for a follow up improvement)